R/stack.R

```diff
@@ -531,6 +530,42 @@ getInfoVar <- function(text, type='Warning: the variable value is just an info f
   return(var)
 }
 
+#' Get unevaluated variables passed as ellipses
+#' 
+#' Get variables which are passed as ellipses without evaluating them.
+#' @param env The environment where the "..." variable is present
+#' @return A named list for each element of "..." containing the 
+#'   expression that will be evaluated, 
+#'   the environment where it will be evaluated, and the string 
+#'   representation of the expression.
+getDotVars <- function(env) {
+  ## Note: substitute(...()) is officially not supported, but it 
+  ## works as intended for all relevant R versions (at least from R 3.0.0)
+  dots <- substitute(...(), env = env)
```

Comment:
Or, as suggested by @renkun-ken:
```r
dots <- eval(quote(match.call(expand.dots = FALSE)), env)[["..."]]
```

Reply:
Yes, it looks less "tricky" but is known to more.